src/Shared/Recruit.Shared.Web/Extensions/EnumExtensions.cs

```diff
@@ -19,8 +19,10 @@ public static string GetDisplayName(this Enum enumValue)
 
         private static readonly Dictionary<Enum, string> DisplayNames = new Dictionary<Enum, string>
         {
+#pragma warning disable CS0618 // Type or member is obsolete
             { ProgrammeLevel.FoundationDegree, "Foundation Degree" },
             { ProgrammeLevel.Masters, "Master's Degree" },
```

Comment:
Why can't we remove this if nothing is going to be set to this? Or is this after your first round of changes?

Reply:
I'm disabling compiler warnings for a known violation. I don't like compiler warnings, ideally we should treat warnings as errors.

Reply:
You can put in treat warnings as errors, so long as you put it across all projects

Reply:
I know, and we haven't done that, but that's no reason to allow warnings if they can be avoided.

Reply:
Which is the reason to dump this code, particularly in this class where it's for a UI concern.